CONTRIBUTING.md

```diff
@@ -73,7 +79,9 @@ clang-format -i src/*.{h,c}
 But, it's better to use git hooks.
 
 ## Git Hooks
-Refloat uses `lefthook` for managing git hooks. Again, you need to install this dependency in your development environment. Then run this to install the commit hook in the repo:
+Refloat uses `lefthook` for managing git hooks. Again, you will need to provide this dependency yourself. See: [`lefthook` on NPM](https://www.npmjs.com/package/lefthook)
```

Comment:
Why the link to npm? It's a hipster js thing, not everyone is a hipster ok? :grin: On a more serious note, maybe a link to the github page would be better, it lists the various installation methods as well. I install lefhook from my distro.

Reply:
Updated with a link to the GitHub.